src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.model;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Operation {
+    public static final Map<String, Type> TYPE_MAP = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String, Type> typeMap = new HashMap<>();
```
"Map is not deeply immutable "
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names